gap/congruences/congpairs.gi

```diff
@@ -442,8 +477,27 @@ function(cong1, cong2)
 end);
 
 InstallMethod(\in,
-"for dense list and finite semigroup congruence by generating pairs rep",
-[IsDenseList, IsFiniteCongruenceByGeneratingPairsRep],
+"for dense list and enumerable semigroup congruence",
+[IsDenseList, IsEnumerableSemigroupCongruence],
+function(pair, cong)
+  local S;
+  S := Range(cong);
+  if Size(pair) <> 2 then
+    ErrorNoReturn("Semigroups: \\in (for a congruence): usage,\n",
+                  "the first arg <pair> must be a list of length 2,");
+  elif not (pair[1] in S and pair[2] in S) then
+    ErrorNoReturn("Semigroups: \\in (for a congruence): usage,\n",
```

Comment:
This is a good point, and one James and I have talked about before.  There's a good case for `\in` just returning `false`, but we decided at some point that it would be better to have an error message (since the question of "in" doesn't really make sense if it's not a pair in the semigroup).  For consistency with congruences in the rest of the package, I've continued to implement `\in` in this way here.